news/913.plugin

```diff
@@ -0,0 +1 @@
+Revert the use of "_" to separate the path element in keys for the Ax sweeper plugin.
```

Comment:
This is a breaking change. use api_change file extension (see pyproject.toml for available news types).